src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -216,7 +216,7 @@ public void Add_PrefixAlreadyRegisteredAndStarted_ThrowsHttpListenerException()
 
         [Theory]
         [InlineData("http://microsoft.com/")]
-        [InlineData("http://[]/")]
+        //Managed implementation allows this. Is this a bad test? [InlineData("http://[]/")]
```

Comment:
Obviously I'm unsure about these.
Windows: throws HttpListenerException
Managed: doesn't throw anything

Should I just remove these tests? It seems like this doesn't actually test anything specific to the S.N.HttpListener implementations, but rather a difference between native Windows APIs and Sockets